package.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Should we remove this file?

Reply:
As, both client and server has seperate installations, I dont think this is required.